bin/test.py

```diff
@@ -289,7 +397,9 @@ def test_stack(session, config, stack_type):
     print 'Deleting stacks'
     for stack in stacks:
         for stack_id in stack['stack_ids']:
-            print 'Deleting stack: {} - region: {}'.format(stack_id, stack['region'])
```

Comment:
Need a cleanup fargate function. Need to empty the CFN bucket and ECR repository.

Reply:
Added cleanup function.